common/ECSResourcesCheck/ReadMe.md

```diff
@@ -0,0 +1,76 @@
+#Automatic check and comparison of ECS cluster
+**How to use**
```

Comment:
Please specify how to run the command-line program, what to pass in, which parameters are required or optional.

Reply:
done